witwidget/notebook/jupyter/js/lib/wit.js

```diff
@@ -25,7 +25,7 @@ var WITView = widgets.DOMWidgetView.extend({
     this.loadAndCreateWhatIfToolElement();
 
     // Add listeners for changes from python.
-    this.model.on('change:examples', this.examplesChanged, this);
+    this.model.on('change:examples_batch', this.appendStagedExamples, this);
     this.model.on('change:config', this.configChanged, this);
     this.model.on('change:inferences', this.inferencesChanged, this);
```

Comment:
need to add same batching logic to inferences (indices, per-model inference results, and per-model extra outputs such as attributions). see similar logic in merged colab version of this PR (https://github.com/PAIR-code/what-if-tool/pull/120)